db/collab.go

```diff
@@ -28,6 +34,29 @@ func init() {
 	sessions = make(map[string]Session)
 }
 
+func (s *Session) AddConn(uid string, conn *websocket.Conn) error {
+	s.Lock.Lock()
+	s.Conns[uid] = conn
+	s.Lock.Unlock()
+	return nil
+}
```

Comment:
Opportunity to return an error here. What if the given UID is already associated to a connection?